src/main/java/frc/robot/Align.java

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.interfaces.Gyro;
+
+import org.opencv.core.Mat;
+
+import edu.wpi.first.wpilibj.ADXRS450_Gyro;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.ADXRS450_Gyro;
+
+/** Add your docs here. */
+public class Align {
+    Apriltags aprilTags = new Apriltags();
+    ADXRS450_Gyro gyro = new ADXRS450_Gyro();
+
+    // public void Alignment() {
+    //     // Set to top pipeline
+    //     apriltags.setPipeline(1);
+    //     Robot.drive.angleAlign();
+    //     // Set to bottom pipeline
+    //     apriltags.setPipeline(2);
+
+    //     // Convert continous angle to limited to 360
+    //     double currentPosition = gyro.getAngle();
+    //     if (gyro.getAngle() > 360) {
+    //         int number = (int) (gyro.getAngle() / 360); 
+    //         currentPosition = gyro.getAngle() - (360 * number);
+    //     }
+    //     if (gyro.getAngle() < 0) {
+    //         int number = (int) (gyro.getAngle() / 360); 
+    //         currentPosition = gyro.getAngle() + (360 * number);
+    //     }
+
+    //     // Calculations for side to side distance.
+    //     double totalAngle = 90 - currentPosition;
+    //     double angle2 = 180 - 90 - totalAngle;
+    //     double angle3 = totalAngle - apriltags.getHorizontalDegToTarget();
+    //     double angleOffset = apriltags.getHorizontalDegToTarget() * Math.PI/180;
+    //     double angle2radians = angle2 * Math.PI/180;
+    //     double angle3radians = angle3 * Math.PI/180;
+
+    //     double length = 17 * Math.sin(angle2radians) / Math.sin(angleOffset);
+    //     double distance = Math.cos(angle3) * length;
+
+    //     // Actual movement
+    //     Robot.drive.turnRobot.drive(5000, totalAngle, 20);
+    //     Robot.drive.encoderRobot.drive(30, distance, "forward", 5000);
+    //     Robot.drive.turnDrive(5000, -90, 20);
+
+    // }
```

Comment:
Not sure if we're planning to use this method for the final robot? If not, I'd recommend deleting here since it'll be saved in git hub history :)